youtube_dl/options.py

```diff
@@ -818,6 +818,10 @@ def _comma_separated_values_options_callback(option, opt_str, value, parser):
         '--add-metadata',
         action='store_true', dest='addmetadata', default=False,
         help='Write metadata to the video file')
+    postproc.add_option(
+        '--preferred-info',
+        dest='preferredinfo', metavar='FILE',
+        help='Override metadata on the outputted file')
```

Comment:
No, that won't work, nobody will bother crafting JSON file just to just override single info meta field. There should be interface for providing overrides for a single meta field from command line in the first place.

Reply:
@dstftw I'll add functionality for individual metadata overrides in additional to supporting a file based JSON override